Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -53,6 +53,7 @@
     'StorageManager',
     'AP_Tuning',
     'AP_RPM',
+    'AP_EFI',
```

Comment:
this list is mostly alphabetical, would be nice to have this listed before AP_GPS